ext/bin/pdbbox-init

```diff
@@ -176,4 +176,40 @@ ssl-key = $abs_pdbbox/ssl/pdb.key.pem
 disable-update-checking = true
 EOF
 
+# do some tuning on PostgreSQL
+
+# generally accepted configuration parameters:
+# Set the shared_buffers to be 1/4 of the amount of RAM
+
+# for development only (considered unsafe in production):
+# Turn off fsync
+# Turn off synchronous commit
+
+case "$OSTYPE" in
+    darwin*)
+        total_ram_bytes=$(sysctl hw.memsize | awk '{ print $2 }') 
+        total_ram_kb=$(expr $total_ram_bytes / 1024) ;;
+    *)
+        total_ram_kb=$(cat /proc/meminfo | grep MemTotal: | awk '{ print $2 }') 
+esac
```

Comment:
Might not matter, but tempted to include the `;;` termination line for the final clause.

Reply:
done